packages/calcite-components/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,291 @@
+import { Component, Element, Fragment, h, Host, Prop, State, VNode, Watch } from "@stencil/core";
+import { Appearance, Kind, Scale } from "../interfaces";
+import { CSS } from "./resources";
+
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  NumberingSystem,
+  numberStringFormatter,
+} from "../../utils/locale";
+
+@Component({
+  tag: "calcite-meter",
+  styleUrl: "meter.scss",
+  shadow: true,
+})
+export class Meter implements LocalizedComponent {
```

Comment:
- Since it has `disabled` I would think `InteractiveComponent` at minimum. 
- Since it has a `value` I would think maybe its a `FormComponent`?
- If it needs a `setFocus` method then `LoadableComponent`?
- If it can be wrapped in a `calcite-label` then `LabelableComponent`?

Reply:
Yeah, wasn’t sure on these. 

I don’t think it needs ‘disabled’ at all. I’ll remove that property and style.

It also isn’t interactive for a user, so while it can be placed inside a Label, a user can’t input or provide any value, so I’m not sure if it needs the Form / Labelable 🤔 WDYT?

Reply:
hmm not sure. The labelable component also helps to focus the component when the label is clicked with and without the "for" attribute.

Reply:
Component has been updated, I still am not sure Form / Interactive / Labelable apply as the component itself won't accept input or be focusable.

Reply:
Cool. Can add `FormComponent` - WDYT the others?

Reply:
Agreed on not needing to implement the others.

Reply:
Implemented `FormComponent` and related props.